frontend/src/features/projects/spl/tabs/CloseOutFormTab.tsx

```diff
@@ -32,7 +31,6 @@ const CloseOutFormTab: React.FunctionComponent<ICloseOutFormTabProps> = ({
       <CloseOutPurchaseInformationForm isReadOnly={isReadOnly} />
       <CloseOutSaleInformationForm isReadOnly={isReadOnly} />
       <CloseOutFinancialSummaryForm isReadOnly={isReadOnly} />
-      <CloseOutFinancialsForm isReadOnly={isReadOnly} />
```

Comment:
Is this the **Financing Information** section on the **Close Out Form** tab?

Reply:
Yes